gradle/scripts/lib/java-javadoc.gradle

```diff
@@ -2,15 +2,40 @@ import java.security.MessageDigest
 
 def offlineJavadoc = rootProject.hasProperty('offlineJavadoc')
 def javadocCacheDir = new File(gradle.gradleUserHomeDir, 'caches/package-lists')
-def visitedUrls = new HashSet<String>()
+def cacheDirName = 'caches/package-lists'
 
 task cleanJavadocCache(type: Delete, group: 'Build', description: 'Deletes the javadoc cache directory.') {
     delete javadocCacheDir
 }
```

Comment:
Should we remove `cleanJavadocCache` since `gradle clean` removes `build` directory?

Reply:
Yes, if you move it to the `buildDir`, it shouldn't be needed anymore.

Reply:
Modified to clean the root folder's `build` directory